docs/guides/admin/docs/modules/awss3distribution.md

```diff
@@ -82,18 +82,24 @@ Opencast Service Configuration
+| org.opencastproject.distribution.aws.s3.secret.key                   | Your secret key                                                                          |                              | 40 characters                                                                         |
+| org.opencastproject.distribution.aws.s3.endpoint                     | The endpoint to use                                                                      | Default AWS S3 endpoint      | https://s3.service.com                                                                |
+| org.opencastproject.distribution.aws.s3.path.style                   | Whether to use path style access URL                                                     | false / Default AWS S3 style |                                                                                       |
+| org.opencastproject.distribution.aws.s3.distribution.base            | Where the S3 files are available <br>(derived from bucket & region or set by CloudFront) |                              | http://s3-us-west-2.amazonaws.com/example-org-dist <br> or DOMAIN_NAME.cloudfront.net |
+| org.opencastproject.distribution.aws.s3.presigned.url                | Whether to enable presigned URL                                                          | false                        |                                                                                       |
+| org.opencastproject.distribution.aws.s3.presigned.url.valid.duration | Valid duration for presigned URL in ms                                                   | 21600000 (6 hours)           |                                                                                       |
+| org.opencastproject.distribution.aws.s3.max.connections              | Number of max connections                                                                | 50                           |                                                                                       |
+| org.opencastproject.distribution.aws.s3.connection.timeout           | Connection timeout in ms                                                                 | 10000                        |                                                                                       |
+| org.opencastproject.distribution.aws.s3.max.retries                  | Number of max retries                                                                    | 100                          |                                                                                       |
+| job.load.aws.s3.distribute                                           | Distribute job load                                                                      | 0.1                          |                                                                                       |
```

Comment:
So, I hate to be a complainer here, but these aren't exposed on the *other* S3 integration...

Is there a reason you're exposing these?  I don't personally care, but I feel like the chance of someone needing to mess with these values is *so* slim that if they actually *do* need to, then they probably need some kind of larger refactor rather than to mess with the loads.

That being said, if this is something that the sponsor wants I'm cool with it - but it should be in both places :D

Reply:
I didn't expose the job loads, the job loads are already part of the configuration. I just added them to the documentation! Please don't shoot the messenger! :D (I totally agree with you btw, but then, most of the s3 code should probably be shared anyway...)